src/styles/modules/_buttons.scss

```diff
@@ -35,3 +37,13 @@
       color: $action-color-darker;
     }
   }
+
+
+@media screen and (min-width:720px){
```

Comment:
Let's put media queries inside the element, you can also use [@include media](http://thoughtbot.github.io/neat-docs/latest/#media) from neat